src/js/disability-benefits/526EZ/pages/veteranInfo.js

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash';
+
+import currentOrPastDateUI from '../../../common/schemaform/definitions/currentOrPastDate';
+import fullNameUI from '../../../common/schemaform/definitions/fullName';
+import ssnUI from '../../../common/schemaform/definitions/ssn';
+import { genderLabels } from '../../../common/utils/labels';
+
+import { veteranInformationViewField, getVerifiedChapterPair } from '../helpers';
+
+export default function createVeteranInfoChapter(formSchema, isReview) {
+
+  const { fullName, date } = formSchema.definitions;
+
+  const uiSchema = {
+    fullName: fullNameUI,
+    socialSecurityNumber: _.assign(ssnUI, {
+      'ui:title': 'Social Security number',
+      'ui:required': form => !form.vaFileNumber,
+    }),
+    vaFileNumber: {
+      'ui:title': 'VA file number',
+      'ui:required': form => !form.socialSecurityNumber,
+      'ui:errorMessages': {
+        pattern: 'Your VA file number must be between 7 to 9 digits'
+      }
+    },
+    dateOfBirth: currentOrPastDateUI('Date of birth'),
+    gender: {
+      'ui:title': 'Gender',
+      'ui:options': {
+        labels: genderLabels
+      },
+    }
+  };
+
+  const schema = {
+    type: 'object',
+    required: ['fullName', 'socialSecurityNumber', 'vaFileNumber', 'gender', 'dateOfBirth'],
```

Comment:
@goldenmeanie @cvalarida @markgreenburg can you provide guidance on what is required here? The prototype indicates that nothing is required (perhaps due to the prefill), the request model and backend schema don't seem to mention these.

Reply:
Thanks @goldenmeanie , sounds good. We'll make ssn conditionally required if VAFileNumber is not present (and vice versa), according to your comment below.